mllib/src/test/scala/org/apache/spark/ml/linalg/VectorUDTSuite.scala

```diff
@@ -44,4 +47,11 @@ class VectorUDTSuite extends SparkFunSuite {
     assert(dataType.asInstanceOf[StructType].fields.map(_.dataType)
       === Seq(new VectorUDT, DoubleType))
   }
+
+  test("typeof Vector") {
```

Comment:
Why do we need this test? Does it provide more test coverage than the one in `UserDefinedTypeSuite`?

Reply:
Yeah, seems we can remove this one.

Reply:
just add this for this comments https://github.com/apache/spark/pull/26599#issuecomment-555929149, it does no more than `UserDefinedTypeSuite`

Reply:
I didn't mean so and I just said: https://github.com/apache/spark/pull/26599#issuecomment-555804633

Reply:
I am sorry for the misunderstanding, my fault

Reply:
can we remove it in a followup?

Reply:
yea, raised and merged

Reply:
https://github.com/apache/spark/pull/26620